aeneas/src/x86/X86CodeGen.v3

```diff
@@ -596,12 +686,28 @@ class X86CodeGen extends MachCodeGen {
 		}
 		return (i, 1);
 	}
+	def isSigned(i: SsaApplyOp, isReturn: bool) -> bool {
+		if (isReturn) return IntType.!(i.getType().typeCon.singleton).signed;
```

Comment:
You shouldn't need to go to .typeCon.singleton; that just gets you back to the same type.

Reply:
Ah. My bad. Fixed.